libraries/provider_service.rb

```diff
@@ -37,7 +37,8 @@ def action_configure
         mode '0755'
         variables(
           # we need to include something about #{progname} fixed in here.
-          program_name: new_resource.service_name
+          program_name: new_resource.service_name,
+          startup_timeout: node['elasticsearch']['service']['startup_timeout']
```

Comment:
This is referring to the node object, when it should be `new_resource.startup_timeout`, right?